pytest_doctestplus/plugin.py

```diff
@@ -51,6 +51,23 @@ def __exit__(self, *args, **kwargs):
 """.lstrip()
 
 
+@pytest.fixture(autouse=True)
+def _docdir(request):
+    """Run doctests in isolated tmpdir so outputs do not end up in repo"""
+    # Trigger ONLY for doctestplus
+    doctest_plugin = request.config.pluginmanager.getplugin("pytest_doctestplus")
+    if isinstance(request.node, doctest_plugin.DocTestTextfilePlus):
```

Comment:
Not sure how to access the class name when this fixture lives in the plugin itself and not over at core lib.
```
AttributeError: module 'pytest_doctestplus.plugin' has no attribute 'DocTestTextfilePlus'
```